pipeline/frontend/yaml/types/base/deprecations.go

```diff
@@ -0,0 +1,59 @@
+// Copyright 2023 Woodpecker Authors
```

Comment:
```suggestion
// Copyright 2024 Woodpecker Authors
```

as this is a new file.